src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -535,34 +535,49 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             Return MakeEndOfLineTrivia(GetNextChar)
         End Function
 
+        Private Function TryGet(ByRef Ch As Char) As Boolean
+            Dim ok = CanGet()
+            Ch = If(ok, Peek(), Ch)
+            Return ok
+        End Function
+
+        Private Function TryGet(num As Integer, ByRef Ch As Char) As Boolean
```

Comment:
>Ch [](start = 54, length = 2)

nit: `Ch` parameter should be lowercase (here and above)

Reply:
done